resources.html

```diff
@@ -0,0 +1,23 @@
+---
+layout: skinny
+show_newsletter: true
+title: Resources
+---
+
+<section>
+  <p>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Tellus id interdum velit laoreet. Egestas sed sed risus pretium. Porttitor leo a diam sollicitudin tempor id eu. Morbi blandit cursus risus at. Sit amet volutpat consequat mauris nunc congue nisi vitae suscipit. Sed risus pretium quam vulputate dignissim suspendisse in est. Proin nibh nisl condimentum id venenatis a condimentum. Ipsum nunc aliquet bibendum enim. Ultrices sagittis orci a scelerisque purus semper eget duis at. Dignissim sodales ut eu sem integer vitae justo eget. Interdum posuere lorem ipsum dolor sit amet consectetur adipiscing elit. Auctor urna nunc id cursus metus aliquam eleifend mi in. Sit amet commodo nulla facilisi nullam vehicula ipsum. Habitasse platea dictumst quisque sagittis. Enim praesent elementum facilisis leo vel fringilla est ullamcorper. Ligula ullamcorper malesuada proin libero nunc consequat interdum.
+  </p>
+</section>
+
+<section>
+  {% for resource in site.data.resources %}
+    <h2>{{ resource.name }}</h2>
+      <p>{{resource.explainer}}</p>
+      <ul>
+        {% for link in resource.links %}
+          <li><a href={{ link }}>Link</a></li>
```

Comment:
Here let's use the link `href` as the text, rather than repeating `Link` for each one.

Ideally we'll have files that we can parse into meaningful names e.g. `/getting-started/contactless-payments.pdf` in the bucket could be rendered under `Getting Started` with the link text `Contactless Payments`. We can worry about that when we move into automating the GCS pull.

Reply:
Now resolved https://github.com/cal-itp/mobility-marketplace/pull/320/commits/fc75434cf8a375d826d0343ab19df37833932562

<img width="838" alt="image" src="https://user-images.githubusercontent.com/3673236/154166523-28809201-1ceb-4318-a594-a8d6cb94e290.png">